### ISSUE TITLE
Table - ActionsColumn: a11y violation in actions menu when triggering modal

### ISSUE THREAD

**Issue by jenny-s51** [patternfly/patternfly-react#8971]
**Is this a new component or an extension of an existing one?**

ActionsColumn

**Describe the feature**

When testing in ODH, this a11y violation was raised: https://docs.deque.com/issue-help/1.0.0/en/focus-modal-not-returned

If the consumer toggles a modal open via the actions menu kebab, the kebab is closed but it should remain open and the triggering element should retain focus when the modal is closed. 

**The focus MUST be purposely moved or set (via JavaScript) onto the appropriate element when the user's action requires a change of context or location for effective keyboard or touch interaction.**

**The focus MUST NOT become lost or reset to the top of the page, except when loading or re-loading a page.**

TODO: Update the ActionsColumn dropdown logic to support the isOpen state on the kebab when an item is selected.
